RickLeinecker/Fortuna
# SetWagerPopup implemented and cleaned up armory.

**Description**
The armory now allows user's to set the wager and their favorite tank.

Resolves #242 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Go to armory.
2. Click Setup button on left-hand side.
3. Enter amount to wager.
4. Click wager.
5. View the change in your currency.
6. View the indication of your current tank and wager in the bottom left.
7. Check the console and DB for errors.
8. Set another wager.
9. Make sure currency is correctly updated (old wager out, new wager in).

**Final Checklist:**
Go down the list and check them off.
